### PR TITLE
Rework Google authentication workflow

### PR DESCRIPTION
**Pull request by michplunkett**
## Describe your changes
This PR will represent a restructure of how authorization is done for the Google Cloud Platform. Instead of manually handling the credentials from a service account, we are going to make use of the [`gcloud` CLI](https://cloud.google.com/sdk/gcloud) and have it handle the creation of our project and our authentication credentials. We will then access all objects in our Google Cloud project through the use of the [google-cloud-node](https://github.com/googleapis/google-cloud-node) library.

## Checklist before requesting a review
- [x] All features within the repository still work.

```bash
michaelp@MacBook-Air-5 icj-project-rig % gulp
`import sass from 'sass'` is deprecated.
Please use `import * as sass from 'sass'` instead.
[15:55:48] Using gulpfile ~/Documents/GitHub/icj-project-rig/gulpfile.js
[15:55:48] Starting 'default'...
[15:55:48] Starting 'clean'...
[15:55:48] Finished 'clean' after 5.01 ms
[15:55:48] Starting 'styles'...
[15:55:51] Finished 'styles' after 2.53 s
[15:55:51] Starting 'copy'...
[15:55:51] Finished 'copy' after 4.13 ms
[15:55:51] Starting 'lint'...
[15:55:51] Starting 'scripts'...
[15:55:51] Starting 'images'...
[15:55:51] Finished 'lint' after 452 ms
[15:55:51] Finished 'scripts' after 488 ms
[15:55:51] gulp-imagemin: Minified 0 images
[15:55:51] Finished 'images' after 492 ms
[15:55:51] Starting 'nunjucks'...
[15:55:51] Finished 'nunjucks' after 1.78 ms
[15:55:51] Starting 'bake'...
[15:55:51] Finished 'bake' after 1.98 ms
[15:55:51] Finished 'default' after 3.03 s
michaelp@MacBook-Air-5 icj-project-rig % gulp format
`import sass from 'sass'` is deprecated.
Please use `import * as sass from 'sass'` instead.
[15:56:02] Using gulpfile ~/Documents/GitHub/icj-project-rig/gulpfile.js
[15:56:02] Starting 'format'...
[15:56:02] Finished 'format' after 104 ms
michaelp@MacBook-Air-5 icj-project-rig % 
```

### CODE REVIEW

File: tasks/nunjucks.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 import nunjucksRender from 'gulp-nunjucks-render';
 import * as journalize from 'journalize';
 import browserSync from 'browser-sync';
-import config from '../project.config.json' assert {type: 'json'};
 import log from 'fancy-log';
-import fs from 'fs';
+
+const config = fs.readJson('../project.config.json');
+
+const config = fs.readJson('../project.config.json');
```
[the remark this answers is not in the document]
Comment (michplunkett) [reply tasks/nunjucks.js]: It works!

**Review by jefflinwood — APPROVED** [review]
This looks good.

**Review by critmcdonald — APPROVED** [review]
Everything seems to work locally and the Google Authentication changes are very worthy. What I don't see is how we can authenticate while running within Github Codespaces. Was that something you were able to figure out?

**Comment by michplunkett** [comment]
We can, from what I can tell, make it work with something like this: https://stackoverflow.com/a/72022681

**Comment by michplunkett** [comment]
That can be added to the setup docs, if you'd like.

**Comment by michplunkett** [comment]
It'd happen sometime Wednesday night, but it'd be a quick add.

**Comment by michplunkett** [comment]
Re-ran the commands after the merge and everything looks good:
```bash
michaelp@MacBook-Air-5 icj-project-rig % gulp
`import sass from 'sass'` is deprecated.
Please use `import * as sass from 'sass'` instead.
[22:06:52] Using gulpfile ~/Documents/GitHub/icj-project-rig/gulpfile.js
[22:06:52] Starting 'default'...
[22:06:52] Starting 'clean'...
[22:06:52] Finished 'clean' after 4.17 ms
[22:06:52] Starting 'styles'...
[22:06:55] Finished 'styles' after 2.66 s
[22:06:55] Starting 'copy'...
[22:06:55] Finished 'copy' after 3.92 ms
[22:06:55] Starting 'lint'...
[22:06:55] Starting 'scripts'...
[22:06:55] Starting 'images'...
[22:06:55] Finished 'lint' after 442 ms
[22:06:55] Finished 'scripts' after 475 ms
[22:06:55] gulp-imagemin: Minified 0 images
[22:06:55] Finished 'images' after 480 ms
[22:06:55] Starting 'nunjucks'...
[22:06:55] Finished 'nunjucks' after 1.59 ms
[22:06:55] Starting 'bake'...
[22:06:55] Finished 'bake' after 2.11 ms
[22:06:55] Finished 'default' after 3.15 s
michaelp@MacBook-Air-5 icj-project-rig % gulp format
`import sass from 'sass'` is deprecated.
Please use `import * as sass from 'sass'` instead.
[22:07:07] Using gulpfile ~/Documents/GitHub/icj-project-rig/gulpfile.js
[22:07:07] Starting 'format'...
[22:07:07] Finished 'format' after 109 ms
michaelp@MacBook-Air-5 icj-project-rig % 
```

**Pull request merged**